spec/index.bs

```diff
@@ -2530,6 +2522,66 @@ enum GPUBindingType {
 };
 </script>
 
+Each {{GPUBindingType}} has an associated {{GPUBindingResource}} type and [=internal usage=], given
+by this table:
+
+<table class="data">
+  <thead>
+    <tr>
+      <th>{{GPUBindingType}}</th>
+      <th><dfn dfn>Binding resource type</dfn></th>
+      <th><dfn dfn>Binding usage</dfn></th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>{{GPUBindingType/"uniform-buffer"}}</td>
+      <td>{{GPUBufferBinding}}</td>
+      <td>[=internal usage/constant=]</td>
+    </tr>
```

Comment:
just fyi you can omit tr/td/th closing tags if you want (we've been doing it elsewhere as it makes the markup less unwieldy). `<tbody>` is also optional.

Reply:
Done